src/applications/vaos/appointment-list/components/CommunityCareAppointmentDetailsPage.jsx

```diff
@@ -0,0 +1,168 @@
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+import { Link, useHistory, useParams } from 'react-router-dom';
+import LoadingIndicator from '@department-of-veterans-affairs/component-library/LoadingIndicator';
+
+import moment from '../../lib/moment-tz';
+
+import { FETCH_STATUS } from '../../utils/constants';
+import { scrollAndFocus } from '../../utils/scrollAndFocus';
+import * as actions from '../redux/actions';
+import AppointmentDateTime from './cards/confirmed/AppointmentDateTime';
+import { getVARFacilityId } from '../../services/appointment';
+import AppointmentInstructions from './cards/confirmed/AppointmentInstructions';
+import AddToCalendar from '../../components/AddToCalendar';
+import { selectFeatureCancel } from '../../redux/selectors';
+import FacilityAddress from '../../components/FacilityAddress';
+
+function CommunityCareAppointmentDetailsPage({
+  appointmentDetails,
+  appointmentDetailsStatus,
+  fetchConfirmedAppointmentDetails,
+  confirmedStatus,
+}) {
+  const { id } = useParams();
+  const history = useHistory();
+
+  const appointment = appointmentDetails?.[id];
+
+  useEffect(() => {
+    const status = confirmedStatus === FETCH_STATUS.succeeded;
+
+    if (!status) {
+      history.push('/');
+    }
+
+    if (!appointment) {
+      fetchConfirmedAppointmentDetails(id);
+    }
+
+    scrollAndFocus();
+  }, []);
+
+  if (appointmentDetailsStatus === FETCH_STATUS.loading) {
+    return (
+      <div className="vads-u-margin-y--8">
+        <LoadingIndicator message="Loading your confirmed appointments..." />
+      </div>
+    );
+  }
+
+  if (!appointment) {
+    return null;
+  }
+
+  const header = 'Community Care';
```

Comment:
Generally, I think the content folks want these headers to be sentence case (this one and the instructions one below). @outerpress, does that sound right to you? It looks like the mockups use title case.

Reply:
That's right @jbalboni , both "Community care" and "Special instructions" should be sentence case. I'll get that updated in the Sketch file as well.